.eslintignore

```diff
@@ -1,5 +1,3 @@
 /node_modules
 /build
 /public/build
-/postgres-data
-/mysql-data
```

Comment:
Why do we want to lint the sql data folder?

I get weird errors if the sql data folder is linted

![image](https://user-images.githubusercontent.com/55592708/216363231-7fc09e14-0fd3-4c10-9a08-d15d75f4d1cd.png)

Reply:
Same issue with `npm run format`

Reply:
Only js, ts jsx and tsx files should be ever linted. Give me a sec

Reply:
We have other file types in the repo, like markdown and yaml. I think we should be excluding the stuff we dont want to lint / format, rather than including everything we would like to

Reply:
theres also a `.js.map`

Reply:
The issue is, that that folder it references was created by docker, and it cannot scan the folder for js/ts/jsx/tsx files so it fails.
You are right, because of this mysql-data has to be added to eslintignore and prettierignore